React/Common SSR errors.md

```diff
@@ -0,0 +1,62 @@
+# Common SSR errors
+
+## Avoid usage of `:first-child` CSS selector with Emotion 10
+
+If `:first-child` CSS selector is used in SSR application React will show this warning in the console:
+```
+The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type"
+```
+
+Default server side rendering in Emotion 10 renders the `<style>` tag inline with the component, instead of extracting everything inside `<head>`. Similar to "shadow CSS" in Web Component.
```

Comment:
```suggestion
Default server-side rendering in Emotion 10 renders the `<style>` tag inline with the component, instead of extracting everything inside `<head>`, similarly to "shadow CSS" in Web Components.
```